packages/appstore/src/components/onboarding-new/static-dashboard.tsx

```diff
@@ -66,21 +66,19 @@ const StaticDashboard = ({
     const { client, traders_hub } = useStores();
     const { content_flag, CFDs_restricted_countries, financial_restricted_countries } = traders_hub;
     const { is_eu_country, is_logged_in } = client;
-
-    const [index, setIndex] = React.useState<number>(0);
-
-    const Divider = () => <div className='divider' />;
-
     const eu_user =
         content_flag === ContentFlag.LOW_RISK_CR_EU ||
         content_flag === ContentFlag.EU_REAL ||
         content_flag === ContentFlag.EU_DEMO;
-
     const is_eu_user = (is_logged_in && eu_user) || (!is_logged_in && is_eu_country);
 
+    const [index, setIndex] = React.useState<number>(is_eu_user ? 1 : 0);
```

Comment:
why do we need to set the index to 1 for eu-user? could you please explain the logic here?

Reply:
@amina-deriv In the useEffect hook, there is logic to switch between CFD tab and Multipliers tab every 5 seconds (as an animation during the time of onboarding). so if its eu-user, setting the initial value to 1 will ensure that the animation starts with 'CFD', else it starts with 'Multipliers' selected.
![Screenshot 2023-06-13 at 11 04 35 AM](https://github.com/suisin-deriv/deriv-app/assets/122449658/7ea69a32-2312-4858-9be3-9329f18d38be)